### PR TITLE
exposing native services

### PR DESCRIPTION
**Pull request by pwright**
addresses #202

### CODE REVIEW

**Comment by pwright** [comment]
@fgiorgetti  Is this worth including?

File: cli/podman.adoc [hunk 1]
```diff
@@ -246,11 +246,29 @@ $ podman run -d --name hello-world-backend --network skupper quay.io/skupper/hel
...
 ----
 $ skupper expose host hello-world-backend --address backend --port 8080 
 ----
 
+If you are exposing a service that is running on the same host as your site that is not a podman container, you need to specify an external IP address, not `localhost`. 
+
+For example, you can find the currently used IP address with the following command:
+[source, bash]
+----
+$ ip route get 8.8.8.8 | head -1 | awk ' {print $(NF-2) }'
```
Comment (skitt) [inline cli/podman.adoc:260]: No need for `head` :wink::

```suggestion
$ ip route get 8.8.8.8 | awk 'NR == 1 {print $(NF-2) }'
```

**Comment by fgiorgetti** [comment]
@pwright Why not using `host.containers.internal` as the hostname to be exposed?
Podman adds it to the /etc/hosts of each container. It is (possibly) the first IP found 
(excluding loopback), but I believe it is easier to use.

**Comment by pwright** [comment]
> @pwright Why not using `host.containers.internal` as the hostname to be exposed?
@fgiorgetti updated to use your suggestion

**Review by fgiorgetti — APPROVED** [review]

**Pull request merged**